.dockerignore

```diff
@@ -0,0 +1,3 @@
+.gitignore
+.env
+node-modules/*
```

Comment:
Have you considered putting in your entire folder client folder in here? Since it gets bundled its not necessary when u dockerize

Reply:
thats true. good point